ontotext-yasgui-web-component/src/css/_variables.scss

```diff
@@ -0,0 +1,2 @@
+$color-ontotext-red: #e84e0f;
+$onto-blue: #003663;
```

Comment:
Let's be consistent. Rename one of these to match the other

Reply:
done